Formula/libfido2.rb

```diff
@@ -0,0 +1,90 @@
+class Libfido2 < Formula
+  desc "Provides library functionality for FIDO 2.0, including USB connections"
+  homepage "https://developers.yubico.com/libfido2/"
+  url "https://github.com/Yubico/libfido2/archive/1.3.0.tar.gz"
+  sha256 "0b2e3671c4c5d42fd5604a08e45f89f49592b97cf66d7d3bfbc7e6a4d5a0fec7"
+  depends_on "cmake" => :build
+  depends_on "mandoc" => :build
+  depends_on "libcbor"
+  depends_on "openssl@1.1"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "man_symlink_html"
+      system "make", "man_symlink"
+      system "make", "install"
+    end
+    mv prefix/"man", share/"man"
+  end
+
+  # TODO: Are there any built-in tests that we could run?
+  test do
+    # You'll need to change this test if the version flag's output changes
+    version_number_output = shell_output("#{bin}/fido2-token -V 2>&1")
+    assert_equal version.to_s, version_number_output.strip
+    # This test is similar to `fido2-token -L` but reduces the amount of identifying info in logs
+    (testpath/"test.c").write <<-EOF
+    #include <stddef.h>
+    #include <stdio.h>
+
+    #include <fido.h>
+
+    int main(void) {
+      // N.B. puts/printf can fail, but we're not testing that
```

Comment:
Please remove all comments and keep the code as minimal as possible

Reply:
Understood, thanks!